unstructured/metrics/evaluate.py

```diff
@@ -66,24 +87,25 @@ def __post_init__(self):
         self.ground_truths_dir = Path(self.ground_truths_dir).resolve()
 
         # -- auto-discover all files in the directories --
-        self._document_paths = [
+        self._rel_doc_paths = [
             path.relative_to(self.documents_dir) for path in self.documents_dir.rglob("*")
         ]
-        self._ground_truth_paths = [
+        self._rel_ground_truth_paths = [
             path.relative_to(self.ground_truths_dir) for path in self.ground_truths_dir.rglob("*")
         ]
 
     def on_files(
         self,
-        document_paths: Optional[list[str | Path]] = None,
-        ground_truth_paths: Optional[list[str | Path]] = None,
+        relative_document_paths: Optional[list[str | Path]] = None,
+        relative_ground_truth_paths: Optional[list[str | Path]] = None,
```

Comment:
I just wanted to reduce confusion by being more verbose what those arguments actually are: relative to `self.documents_dir` or `self.ground_truths_dir`.

Note that we may want to refactor the logic/design of this class at some point and make it use full paths instead (only if we don't like the design of providing the root dir and discovering files). For me, it all depends, it can stay as it is.